### PR TITLE
[ZDF] added current test case

### PR DESCRIPTION
**Pull request by Wandang**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Other tests were no longer available (404) so I added a new/current one

### CODE REVIEW

**Comment by dstftw** [comment]
Remove 404 test. Also try to find a test that will last longer (this one will expire in about a month).

**Comment by Wandang** [comment]
Hope those changes are satisfactorily

File: youtube_dl/extractor/zdf.py [hunk 1]
```diff
@@ -42,19 +42,16 @@ class ZDFIE(ZDFBaseIE):
...
-            'timestamp': 1477627200,
-            'upload_date': '20161028',
+            'title': 'Die Magie der Farben (2/2)',
+            'description': 'md5:a89da10c928c6235401066b60a6d5c1a',
+            'duration': 2615,
+            'timestamp': 1465021200,
+            'upload_date': '20160604',
         }
-    }, {
-        'url': 'https://www.zdf.de/filme/taunuskrimi/die-lebenden-und-die-toten-1---ein-taunuskrimi-100.html',
```
Comment (dstftw) [inline youtube_dl/extractor/zdf.py]: Keep this test.

File: youtube_dl/extractor/zdf.py [hunk 2]
```diff
@@ -42,19 +42,16 @@ class ZDFIE(ZDFBaseIE):
     _QUALITIES = ('auto', 'low', 'med', 'high', 'veryhigh')
 
     _TESTS = [{
-        'url': 'https://www.zdf.de/service-und-hilfe/die-neue-zdf-mediathek/zdfmediathek-trailer-100.html',
```
Comment (dstftw) [inline youtube_dl/extractor/zdf.py]: Keep this test as `only_matching`.

**Pull request merged**